config/storybook-addon-knobs.d.ts

```diff
@@ -0,0 +1,29 @@
+declare module 'storybook-addon-knobs' {
+  import WrapStory from "../src/components/WrapStory.js";
+
+  interface KnobOption<T> {
+    value: T,
+    type: 'text' | 'boolean' | 'number' | 'object' | 'select' | 'date',
+  }
+
+  interface StoryContext {
+    kind: string,
+    story: string,
+  }
+
+  export function knob<T>(name: string, options: KnobOption<T>): T;
+
+  export function text(name: string, value: string): string;
```

Comment:
Yes, this can be `value: string | null`.

If you want it to be optional as well, the syntax is `value?: string | null`